vnext/Desktop.IntegrationTests/RNTesterIntegrationTests.cpp

```diff
@@ -190,4 +185,21 @@ TEST_CLASS (RNTesterIntegrationTests) {
   TEST_METHOD(AccessibilityManager) {
     TestComponent("AccessibilityManagerTest");
   }
+
+  #pragma region Extended Tests
+
+  TEST_METHOD(Dummy) {
+    auto result = m_runner.RunTest("IntegrationTests/DummyTest", "DummyTest");
+    Assert::AreEqual(TestStatus::Passed, result.Status, result.Message.c_str());
+  }
+
+  BEGIN_TEST_METHOD_ATTRIBUTE(WebSocketBinary)
+  TEST_IGNORE()
```

Comment:
>TEST_IGNORE [](start = 2, length = 11)

Just checking if intentionally disabled?

Reply:
Yes.
It's not decided whether we'll enable this test in this PR.

It would require to either:
- Launch a second node process to run a new WebSocket server.
- Extend the in-process WebSocket server to suppport binary messages.

This may get deferred to a later PR.